cel/env.go

```diff
@@ -432,6 +462,16 @@ func (e *Env) configure(opts []EnvOption) (*Env, error) {
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
This seems...hacky. But okay.

Reply:
Ah, yes, the compile is a bit hacky. Let me just do the checker construction here.